app/scripts/move-popup/dimTalentGrid.directive.js

```diff
@@ -81,5 +81,20 @@
       vm.numColumns = _.max(vm.talentGrid.nodes, 'column').column + 1 - vm.hiddenColumns;
       vm.numRows = vm.perksOnly ? 2 : (_.max(vm.talentGrid.nodes, 'row').row + 1);
     }
+
+    vm.onScopeClick = function(node) {
+      // Add a better check if is a scope node
+      if (node.column === 1) {
```

Comment:
Why don't we add a ScopesService that loads the scopes database, and you can just ask it whether the hash of the node is a scope? We have that info in the JSON.